stream_alert/athena_partition_refresh/main.py

```diff
@@ -386,11 +465,15 @@ def unique_buckets_from_messages(self):
                     LOGGER.debug(record)
                     continue
 
-                buckets.add(record['s3']['bucket']['name'])
+                bucket_name = record['s3']['bucket']['name']
+                # Account for special characters in the S3 object key
```

Comment:
What are these special characters you speak of 🤔  Consider being consistent with [this comment ](https://github.com/airbnb/streamalert/blob/afd19b203d529d03b2e32542889eda3fc6984fe6/stream_alert/rule_processor/payload.py#L230)